benchmarks/viscoelastic_plastic_shear_bands/kaus_2010/kaus_2010_analysis.py

```diff
@@ -0,0 +1,137 @@
+# Based on the following script:
+# benchmarks/viscoelastic_plastic_shear_bands/gerya_2019/gerya_2019_analysis.py
+# Help: python3 kaus_2010_analysis.py --help
+import argparse
+from pathlib import Path
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import vtk as vtk
+from vtk.util import numpy_support
+
+
+
+def main():
```

Comment:
Move the main function to the bottom of the file (above `if __name_ ...`, so that all functions are already defined when the main function is defined (even if this does not matter, it is the recommended style).